pkg/ddc/alluxio/operations/base.go

```diff
@@ -327,6 +328,32 @@ func (a AlluxioFileUtils) Count(alluxioPath string) (fileCount int64, folderCoun
 	return int64(ufileCount), int64(ufolderCount), int64(utotal), err
 }
 
+// file count of the Alluxio Filesystem (except folder)
+// use "alluxio fsadmin report metrics" for better performance
+func (a AlluxioFileUtils) GetFileCount() (fileCount int64, err error) {
+	var (
+		command = []string{
+			"alluxio", "fsadmin", "report", "metrics", "|", "grep", "Master.TotalPaths"}
```

Comment:
Maybe it's better to `grep Master.FilesCompleted` to exclude dirs?

Reply:
Right, https://github.com/fluid-cloudnative/fluid/issues/299#issuecomment-719902572